R/plotScoreHeatmap.R

```diff
@@ -175,6 +175,9 @@ plotScoreHeatmap <- function(results, cells.use = NULL, labels.use = NULL,
         args$color <- viridis::viridis(100)
         args$legend_breaks <- c(0,1)
         args$legend_labels <- c("Lower", "Higher")
+    } else {
+        abs.max <- max(abs(c(min(scores), max(scores))))
```

Comment:
Or `range()` would do the same as `c(min...)`.

Reply:
True.  But I had already adjusted this down in a different (better?) way.  Sorry for the multiple commits.

Reply:
If I were to be pedantic, your latest version is computing `max(abs(scores))` twice. 

The theoretical benefit of `max(abs(range(scores)))` is that it avoids creating a new matrix with `abs(scores)`.

Reply:
Unfortunately, seq() reqs the `from` and `to` to be separate inputs.  So I was thinking my updated method was the shortest code and therefore better.

I'm not sure the efficiency boost of `max(abs(range(scores)))` will matter too much compared to other calcs, but I supposed it can't hurt.  The code is still easily readable either way.